spec/index.adoc

```diff
@@ -2872,6 +2876,10 @@ S with
           controllers = S.controllers[A.canister_id];
           memory_size = Memory_size;
           cycles = S.balance[A.canister_id];
+          low_on_cycles =
+            if S.balance[A.canister_id] < (freezing_limit(S, A.canister_id) + 100 * MAX_CYCLES_PER_MESSAGE)
```

Comment:
> canister.balance < canister.freezing_threshold_cycles + N * MAX_CYCLES_PER_MESSAGE 

I chose N to be 100, but it might be too low. Happy to hear other proposals.

MAX_CYCLES_PER_MESSAGE does not have a concrete value defined in the spec, but it is used to show that the balance is approaching the freezing threshold.